crates/test-files/fixtures/differential/math_i8.sol

```diff
@@ -16,10 +16,6 @@ contract Foo {
     return val1 / val2;
   }
 
-  function pow(int8 val1, uint8 val2) public pure returns (int8) {
-    return int8(uint8(val1) ** val2);
```

Comment:
I removed this because we can not really test it. Solidity doesn't allow the base to be negative whereas in Fe `-2 **3 = -8` is fine. I initially thought I could cast the problem away but that turned out to be silly and doesn't work. We still perform that test on the `math_u8` tests though.